about/release-notes/index.md

```diff
@@ -17,6 +17,76 @@ GitHub and be notified by email whenever a new release is available. On the
 click `Watch`, select `Custom` and then check `Releases`.
 </Highlight>
 
+## TimescaleDB&nbsp;2.13.0 on 2023-11-28
+
+<Highlight type="note">
+This release contains performance improvements, an improved hypertable DDL API
+and bug fixes since the 2.12.2 release. We recommend that you upgrade at the next
+available opportunity.
+
+In addition, it includes these noteworthy features:
+
+* Full PostgreSQL 16 support for all existing features
+* Vectorized aggregation execution for sum()
+* Track chunk creation time used in retention/compression policies
+</Highlight>
+
+#### Deprecation notice: Multi-node support
+TimescaleDB 2.13 is the last version that will include multi-node support. Multi-node
+support in 2.13 is available for PostgreSQL 13, 14 and 15. Learn more about it
+[here](docs/MultiNodeDeprecation.md).
```

Comment:
This is a link towards our main repo, the file does not live in the docs site

```suggestion
[here](https://github.com/timescale/timescaledb/blob/main/docs/MultiNodeDeprecation.md).
```

Reply:
Good catch, changed